src/web/topic/components/TopicWorkspace/MoreActionsDrawer.tsx

```diff
@@ -345,32 +347,36 @@ export const MoreActionsDrawer = ({
             <ListItem disablePadding={false}>
               <Typography variant="body2">Layout Thoroughness</Typography>
               <Tooltip
-                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. 1 = lowest effort, 100 = highest effort."
+                title="Determines how much effort the layout algorithm puts into laying out nodes such that they efficiently use space. Low = minimal effort, High = maximum effort."
                 enterTouchDelay={0} // allow touch to immediately trigger
                 leaveTouchDelay={Infinity} // touch-away to close on mobile, since message is long
               >
                 <IconButton
                   color="info"
                   aria-label="Thoroughness info"
                   sx={{
-                    // Don't make it look like clicking will do something, since it won't.
-                    // Using a button here is an attempt to make it accessible, since the tooltip will show
-                    // on focus.
-                    cursor: "default",
+                    cursor: "default", // Maintain accessibility while preventing click actions
                     alignSelf: "center",
                   }}
                 >
                   <Info />
                 </IconButton>
               </Tooltip>
-              <NumberInput
-                min={1}
-                max={100}
+              <Select
```

Comment:
Like with the watch dropdown, I would have used Mui's `TextField` with the `select` prop, because it uses a Mui `Select` component under-the-hood but comes with `TextField`'s nice labeling, like this:

![image](https://github.com/user-attachments/assets/4c684d7b-cde5-4d2d-9d30-d2ed3c10962c)

I think using `Select` here can work too, but then perhaps we could just use "Thoroughness" in the text label, so that the label doesn't go onto two lines ("Layout" I think is implied by being in the "Layout Config" section anyway).